docs/src/algorithms.md

```diff
@@ -0,0 +1,208 @@
+```@setup base
+using Random: seed!
+seed!(42)
+using KernelFunctions
+using Plots
+using InducingPoints
+D = 2
+N = 50
+M = 10
+x = [rand(D) for _ in 1:N]
+N₂ = 25
+x₂ = [rand(D) for _ in 1:N₂] .* 0.25
```

Comment:
How about so that it looks really continuous and we can see changes with `UniGrid` (when it works)
```suggestion
x = [rand(D) .* [0.8, 1.0] for _ in 1:N]
N₂ = 25
x₂ = [rand(D) .* [0.2, 1.0] + [0.8, 0.0] for _ in 1:N₂]
```

Reply:
Sure, I think that would work. Let me try.  It took a bit of tweaking to get two sets that resulted in changes for in update using `OIPS`.

Reply:
I think it's because the kernel lengthscale is too low. Either, use a higher one or just expand the box where the data is

Reply:
Yeah, I tweaked the length scale a bit, and went for the "smallest" one that showed a difference. .